client/ayon_core/tools/publisher/models/publish.py

```diff
@@ -1219,9 +1232,21 @@ def _process_and_continue(
         plugin: pyblish.api.Plugin,
         instance: pyblish.api.Instance
     ):
-        result = pyblish.plugin.process(
-            plugin, self._publish_context, instance
-        )
+        handler = MessageHandler()
```

Comment:
Should we create this handler once and store it on the class instance instead of iterating every plug-in for every instance? Preferably this iterator does the least amount of things to keep it as fast as possible.

Reply:
Well, technically we can, but at the end it does not matter that much.